tests/test_losses.py

```diff
@@ -73,39 +73,6 @@ class NSSALossTests(cases.SetwiseLossTestCase):
 class TestCustomLossFunctions(unittest.TestCase):
     """Test the custom loss functions."""
 
-    def test_negative_sampling_self_adversarial_loss(self):
```

Comment:
It probably makes sense to instead compare against a reference file e.g. from the latest release.

Reply:
I drafted something here, but I am not too happy with it: https://github.com/pykeen/pykeen/pull/1530

Reply:
#1530 has been merged